bigbluebutton-html5/private/config/settings.yml

```diff
@@ -574,6 +574,7 @@ public:
     traceSip: false
     # SDP semantics: plan-b|unified-plan
     sdpSemantics: 'unified-plan'
+    disableWebsocketFallback: false
```

Comment:
This doesn't look the proper object for this config to be in (`public.media.disableWebsocketFallback`). Maybe you meant public.app.*?

Reply:
That's right, thanks!